packages/runtime/src/next/components/tests/controls/__snapshots__/checkbox-control.test.tsx.snap

```diff
@@ -0,0 +1,603 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Page can render CheckboxControl v0 data 1`] = `
+{
+  "asFragment": [Function],
+  "baseElement": .emotion-0 {
+  position: relative;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.emotion-0>* {
+  border-radius: inherit;
+  height: inherit;
+}
+
+.emotion-0>:last-child {
+  position: relative;
+}
+
+.emotion-1 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-flex-wrap: wrap;
+  -webkit-flex-wrap: wrap;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  width: 100%;
+}
```

Comment:
I'm having some doubts about using snapshot testing given that the snapshot must be a full match. I don't want to cause people headaches any time the Page component changes and our snapshots are stale. Are we sure that the `screen` testing not sufficient? I understand that changing snapshots is a red flag, but I would argue that modifying older tests should raise the same alarm bells.

Reply:
Yeah, I didn't mean to suggest that we snapshot the DOM, there is just too much noise here.